mempool/src/shared_mempool.rs

```diff
@@ -168,12 +171,12 @@ pub struct TransactionExclusion {
 
 /// Mempool's subscription to reconfiguration events
 pub struct MempoolReconfigSubscription {
-    sender: futures::channel::mpsc::Sender<usize>,
+    sender: libra_channel::Sender<usize, ContractEvent>,
```

Comment:
you should use `libra_channel::Sender<(), ContractEvent>`, since there is no "key" you're segmenting the queues on.

Reply:
I was wondering how `libra_channel` handles single-key sending :)